bbot: add package

D3vil0p3r opened the pull request:
Close https://github.com/BlackArch/blackarch/issues/4170

D3vil0p3r commented:
@noraj sorry again for stalking you. On this, I didn't use the [Python standalone PKGBUILD BA template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/python/PKGBUILD-python-standalone) because I got some issues, but I used a different method to use the virtualenv. Let me know if it is ok for you.

noraj commented:
Why is it on draft?

noraj commented:
> I just converted bbot PKGBUILD to a virtual environment approach. I would still keep the other PKGBUILDs because useful for users.

where is the virtualenv version then? also there is no point in keeping this non virtualenv version if it's not working

D3vil0p3r commented:
> > I just converted bbot PKGBUILD to a virtual environment approach. I would still keep the other PKGBUILDs because useful for users.
> 
> where is the virtualenv version then? also there is no point in keeping this non virtualenv version if it's not working

Ja. It should be in the older commit https://github.com/BlackArch/blackarch/pull/4382/commits/1ed1d037e106c38e676424a06fd3ff7ff01dd3b0#diff-4052d14687365b012298fd93b3ccfe8e99e6e71db7756c806cd9b188ccb49975 (on the left side). I removed it because, don't tell me why (I didn't go deep), if I used the previous virtual env approach, it continues to refer to `/tmp/makepkg` directory.
